plugins/dev-tools/src/playground/index.js

```diff
@@ -191,6 +193,7 @@ export function createPlayground(
           currentTab = tab;
           currentGenerate = tab.generate;
           const isXml = tab.state.name == 'XML';
+          const isJson = tab.state.name == 'JSON';
           editor.setModel(currentTab.state.model);
           editor.updateOptions({
             readOnly: !isXml,
```

Comment:
Is the JSON tab supposed to be readonly?

Reply:
Nope! Thanks for the catch!